### DISCUSSION TITLE
How about considering an APY that gives preferential treatment to stakers who develop OSS that contributes to the DEV economic zone through a review system?

### DISCUSSION THREAD

**Discussion by hideakiimoto** [dev-protocol/DIPs#57] [General]
There is a difference in contribution to DEV between HiDEs that encourage redistribution of DEV tokens and creators that just sell the rewards they get as creator rewards. Staking is the best way to use DEV. However, in the current situation, if the reward for large creators is large enough, the selling pressure will prevail and it will be difficult to maintain the price.
If APYs were given to supporters who are staking creators who contribute to the value of DEV, I think that would change the buying pressure.

**Comment by aggre** [comment]
To generalize that review system, I think there are two approaches:
A. Adjust the creator rewards APY according to the manifestation of the staker's intention
B. Adjust the withdrawal limit for creator rewards according to the manifestation of the staker's intention

In either case, I think the adjustment vector should always be "negative" to eliminate the risk of fraud by creators impersonating stakers.

Please tell me your opinion.

**Reply by hideakiimoto** [reply]
#38 made a lot of sense to me after reading it. Thank you very much.

**Reply by aggre** [reply]
The purpose of #55 is to suppress self-staking, which is essentially different from the purpose of this discussion. #55 treats "contributors" and "non-contributors" equally.

I shared the idea of adjusting rewards by stakers. Still, I think it's natural tokenomics to have less staking for projects that you think aren't desirable—more staking for projects that you think are desirable. There is also a problem that depends on the high gas fees, but L2 can solve it. Against this background, the combination of DIP55 and L2 may be the simplest solution.

**Comment by Akira-Taniguchi** [comment]
If it becomes possible to set the ratio of remuneration between creators and investors for each PROPERTY, I have to be careful because investors tend to be attracted to places where the ratio of investors is high.

**Reply by hideakiimoto** [reply]
I think that's what's needed; can we have a product that encourages people to buy DEVs and a product that only sells reward DEVs on the same evaluation?

**Reply by Akira-Taniguchi** [reply]
It is difficult to determine whether it is a sale or a distribution, so I think what we can consider now is the ability to grant restrictions on withdrawing rewards for people staking to you, as represented by DIP38.

https://github.com/dev-protocol/DIPs/pull/38

**Reply by hideakiimoto** [reply]
Ahhh, so this is what was proposed. I understand it quite well. But the restriction to creator rewards seems to work better. Thank you very much.

**Reply by yumemayu** [reply]
We've discussed #38 with your team before. Thanks for checking!

**Reply by aggre** [reply]
DIP #38 is now being replaced by DIP #55.

**Comment by yumemayu** [comment]
How to use creator fees varies from person to person.
There has been a DIP6 before that results in a different APY depending on the creator. What do you think about this? 

#6 Reward split by Property

**Reply by hideakiimoto** [reply]
I think it is fine to pay creators the same.
Each is a precious development.
What I wanted to say was that if support is given to products that do not contribute as much to DEVs as those that do, the incentive to buy DEVs will be eroded in the future and it will be difficult to maintain prices. However, I understood that this can be mitigated by limiting the creators' sales in #38. However, whether the products that contribute to the motivation to purchase DEVs will necessarily be chosen for staking depends on the promotion.

**Comment by defi-er** [comment]
Thank you for making this proposal.

Dev Protocol's first product, Inflationary Funding, is meant to support Creators that sell.  However, when Creators sell DEV there's a negative impact on the price. In the short term the negative impact is decreasing price. In the long term, if the DEV is used to grow the OSS project, then DEV holders benefit by holding 5% of that team's tokens.

I prefer to encourage positive incentives instead of punishing Creators for Dev Protocol's value proposition. There are a number of positive incentives that will encourage Creators to not sell their DEV.

1. Stake DEV for your dependencies to support them and therefore your own product
2. Stake DEV for other Creators to increase your karma
3. Put vesting periods in place that are based on a Creator's karma. If a Creator wants more DEV to withdraw, then they must stake for other Creators. That means that less DEV is sold, more Creators are supported, and Creators make a sustainable community by becoming their own Patrons.
4. DIP #52 Redistributes a percentage of the Social Token Fee (5%) to the Patrons of that project. This means that Patrons will receive a future economic value for the Creators they support. This encourages Patrons and Creators to not sell their DEV, find amazing projects, and invest in them by staking DEV. This is amazing because now Creators can use their industry knowledge to become investors, without burning money, by staking DEV.

There's not one solution for incentivization models. Each person has their own incentivizes that change over time. Therefore, a multiple faceted approach that interlocks to support each other should be approached.
